scrapy/core/scheduler.py

```diff
@@ -61,6 +82,7 @@ def enqueue_request(self, request):
             self._mqpush(request)
             self.stats.inc_value('scheduler/enqueued/memory', spider=self.spider)
         self.stats.inc_value('scheduler/enqueued', spider=self.spider)
+        self.stats.inc_value('scheduler/enqueued/key/{}'.format(self.request_key(request)))
```

Comment:
Hm, I like an extra debugging options in stats, but we may have a problem if a spider is to download e.g. pages from 100K domains, one per domain - stats would become huge and unreadable.

Reply:
Removed.